presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -78,7 +82,7 @@ Array Functions
 .. function:: array_has_duplicates(array(T)) -> boolean
 
     Returns a boolean: whether ``array`` has any elements that occur more than once.
-    Throws an exception if any of the elements are rows or arrays that contain nulls. 
+    Throws an exception if any of the elements are rows or arrays that contain nulls.
```

Comment:
This is non-standard. In sql we don't throw exceptions. You should define a clean null semantics. @rschlussel - can you take a look?

Reply:
These seems to be unrelated. Only removing the spaces.